scipy/sparse/tests/test_array_api.py

```diff
@@ -58,6 +58,18 @@ def test_mean(A):
         "Expected array, got matrix"
 
 
+@parametrize_sparrays
+def test_min_max(A):
+    assert not isinstance(A.min(axis=1), np.matrix), \
+        "Expected array, got matrix"
+    assert not isinstance(A.max(axis=1), np.matrix), \
+        "Expected array, got matrix"
+    assert not isinstance(A.argmin(axis=1), np.matrix), \
+        "Expected array, got matrix"
+    assert not isinstance(A.argmax(axis=1), np.matrix), \
+        "Expected array, got matrix"
```

Comment:
Looks like a subset of the sparse types/objects tested in the parametrized cases don't have `min()` and `max()` and so on, while others do.

This seems to fail as intended on `main` and pass here (I suppose a `try`/`except` on `AttributeError` is also possible, though I was slightly nervous this might allow an unanticipated error to get suppressed).

```diff
--- a/scipy/sparse/tests/test_array_api.py
+++ b/scipy/sparse/tests/test_array_api.py
@@ -60,14 +60,18 @@ def test_mean(A):
 
 @parametrize_sparrays
 def test_min_max(A):
-    assert not isinstance(A.min(axis=1), np.matrix), \
-        "Expected array, got matrix"
-    assert not isinstance(A.max(axis=1), np.matrix), \
-        "Expected array, got matrix"
-    assert not isinstance(A.argmin(axis=1), np.matrix), \
-        "Expected array, got matrix"
-    assert not isinstance(A.argmax(axis=1), np.matrix), \
-        "Expected array, got matrix"
+    if hasattr(A, "min"):
+        assert not isinstance(A.min(axis=1), np.matrix), \
+            "Expected array, got matrix"
+    if hasattr(A, "max"):
+        assert not isinstance(A.max(axis=1), np.matrix), \
+            "Expected array, got matrix"
+    if hasattr(A, "argmin"):
+        assert not isinstance(A.argmin(axis=1), np.matrix), \
+            "Expected array, got matrix"
+    if hasattr(A, "argmax"):
+        assert not isinstance(A.argmax(axis=1), np.matrix), \
+            "Expected array, got matrix"
```

Reply:
Ah, good point. Eventually we should maybe define a `parameterize_data_sparrays`, but this works for now.